docs/user/interactive_basics.rst

```diff
@@ -0,0 +1,175 @@
+.. _interactive_basics:
+
+
+Interactive Environment Basics
+==============================
+
+What is an Interactive Environment?
+-----------------------------------
+
+Interactive environments on RenkuLab are web-based user interfaces (like JupyterLab
+and RStudio) that you can launch to develop and run your code and data workflows.
+They're commonly used for exploratory analysis because you can try out short blocks
+of code before combining everything into a (reproducible) workflow.
+
+You can run your own interactive environments independently from RenkuLab, but
+on RenkuLab, you get the bonus capabilities:
+
+* a configurable amount of resources (memory, CPU, and sometimes GPU) is
+  available on launch
+
+* you can save your work from within the environment back to RenkuLab
+
+* it's a shareable, reproducible, :ref:`customizable<customizing>` container defined by the ``Dockerfile``
+
+* you have all the functionality provided by the renku-python_ command-line interface (CLI)
+
+
+What's in my Interactive Environment?
+-------------------------------------
+
+* your project, which is cloned into the environment (but by default *without
+  files that are stored in git LFS*)
+* all the default software required to launch the environment and common software
+  for code development (``git``, ``git LFS``, ``vim``, etc.)
+* any software you added via ``requirements.txt``, ``environment.yml``, ``install.R``,
+  or directly into the ``Dockerfile``
+* the renku command-line interface renku-python_.
+* the amount of CPUs, memory, and (possibly) GPUs that you configured before launch
+
+For adding or changing software installed into your project's interactive environment,
+check out :ref:`customizing`
+
+
+Which Interactive Environment will launch?
+------------------------------------------
+
+The template you choose when you create a project on RenkuLab (or locally call
+``renku init`` on your project) determines the kind of interactive environment
+that is available to launch. Here's a guide to the templates provided:
+
+* ``Basic Python Project``: JupyterLab
```

Comment:
This is out of date, and is likely to easily get out of date. I think we should just say that there are templates for typical python, R, and Julia projects and point users to a README that describes the template (in the renku-python project, or wherever they are).